alexBot/cogs/video_dl.py

```diff
@@ -87,17 +88,23 @@ async def on_message(self, message: discord.Message):
             return
 
         matches = None
-
-        for regex in REGEXES:
+        convert_reddit = False
+        
+        for i,regex in enumerate(REGEXES):
             matches = regex.match(message.content)
             if matches:
+                convert_reddit = i == 3
                 break
 
         if matches is None:
             return
 
         match = matches.group(0)
         log.info(f'collecting {match} for {message.author}')
+        
+        if convert_reddit:
+            resp = requests.get(url=match + '.json', headers={'User-Agent': 'AlexBot/0.0.1'})
```

Comment:
should probably use aiohttp instead of requests

see https://github.com/mralext20/alex-bot/blob/main/alexBot/cogs/sugery.py#L52 for example of using the get_json helper from ..tools